frontend/design-system/src/index.css

```
@@ -1 +1,20 @@
 @import "tailwindcss";
+
+@font-face {
+  font-family: InterVariable;
+  font-weight: 100 900;
+  font-display: swap;
+  src: url(https://rsms.me/inter/font-files/InterVariable.woff2) format("woff2");
```

Stylelint (17.4.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)